core-bundle/src/DependencyInjection/ContaoCoreExtension.php

```diff
@@ -178,8 +178,11 @@ private function setPredefinedImageSizes(array $config, ContainerBuilder $contai
 
         $imageSizes = [];
 
-        foreach ($config['image']['sizes'] as $name => $value) {
-            $imageSizes['_'.$name] = $this->camelizeKeys($value);
+        // Do not add a size with the special name '_default' but merge its values into all other definitions instead.
+        foreach (array_diff_key($config['image']['sizes'], ['_defaults' => null]) as $name => $value) {
```

Comment:
```suggestion
        foreach ($config['image']['sizes'] as $name => $value) {
            if ('_defaults' === $name) {
                continue;
            }
```

Reply:
applied in a743c70f823beffcf5ab941091bd097051473a2a